### ISSUE TITLE
v1.2.2 `opam install` crashes if something is piped into yes/no prompt; mistakes piped stdin for tty

### ISSUE THREAD

**Issue by qrntz** [ocaml/opam#2966]
Debian/Ubuntu:

```
$ yes | opam install ocsigen-start
...
===== 67 to install =====
Do you want to continue ? [Y/n] Fatal error:
# opam-version    1.2.2 (58ef3b8213100953848d362f7120a30356d7f77d)
# os              linux
opam: "tcgetattr" failed: Inappropriate ioctl for device
```
Alpine:

```
$ yes | opam install ocsigen-start
...
===== 67 to install =====
stty: standard input: Not a tty
Do you want to continue ? [Y/n] Fatal error:
# opam-version    1.2.2
# os              linux
opam: "tcgetattr" failed: Not a tty
```

**Comment by AltGr** [comment]
Thanks for reporting. This fixed already in the 2.0 branch ; it is preferable to use `opam install --yes`, though.

**Comment by qrntz** [comment]
OK, thanks. 1.2.2 is the most recent version in the (stable) Docker images.
